docs/tutorials/10_lattice_models.ipynb

```diff
@@ -894,50 +902,99 @@
     "print(ham)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## LatticeModelProblem\n",
+    "We also have the Lattice Model Problem to generate the Second quantized operator from a Lattice Model. We can use the Ground State Eigensolver with it to calculate the ground state energy."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 23,
    "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.problems.second_quantization import LatticeModelProblem\n",
+    "from qiskit_nature.converters.second_quantization import QubitConverter\n",
+    "\n",
+    "num_nodes = 4\n",
+    "boundary_condition = BoundaryCondition.OPEN\n",
+    "line_lattice = LineLattice(num_nodes=num_nodes, boundary_condition=boundary_condition)\n",
+    "\n",
+    "fhm = FermiHubbardModel.uniform_parameters(\n",
+    "    lattice=line_lattice,\n",
+    "    uniform_interaction=t,\n",
+    "    uniform_onsite_potential=v,\n",
+    "    onsite_interaction=u,\n",
+    ")\n",
+    "\n",
+    "lmp = LatticeModelProblem(lattice_model=fhm)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/html": [
-       "<h3>Version Information</h3><table><tr><th>Qiskit Software</th><th>Version</th></tr><tr><td><code>qiskit-terra</code></td><td>0.19.0.dev0+fb934b7</td></tr><tr><td><code>qiskit-aer</code></td><td>0.9.0</td></tr><tr><td><code>qiskit-ignis</code></td><td>0.7.0</td></tr><tr><td><code>qiskit-nature</code></td><td>0.3.0</td></tr><tr><th>System information</th></tr><tr><td>Python version</td><td>3.8.12</td></tr><tr><td>Python compiler</td><td>Clang 13.0.0 (clang-1300.0.29.3)</td></tr><tr><td>Python build</td><td>default, Nov 12 2021 16:34:26</td></tr><tr><td>OS</td><td>Darwin</td></tr><tr><td>CPUs</td><td>4</td></tr><tr><td>Memory (Gb)</td><td>16.0</td></tr><tr><td colspan='2'>Wed Dec 08 19:13:53 2021 JST</td></tr></table>"
-      ],
-      "text/plain": [
-       "<IPython.core.display.HTML object>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "=== GROUND STATE ===\n",
+      " \n",
+      "* Lattice ground state energy : -2.566350190841\n"
```

Comment:
I am not sure, if this is because of the Github plain-text view of the tutorial, but I don't think this output cell should be here, but rather should be below the next cell